packages/tools/src/groupFilesByType.ts

```diff
@@ -4,22 +4,11 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  *
- * @flow
  */
 
-import {groupBy} from 'lodash';
+import { groupBy } from 'lodash';
 import mime from 'mime';
 
-/**
- * Since there are no officially registered MIME types
- * for ttf/otf yet http://www.iana.org/assignments/media-types/media-types.xhtml,
- * we define two non-standard ones for the sake of parsing
- */
-mime.define({
-  'font/opentype': ['otf'],
-  'font/truetype': ['ttf'],
```

Comment:
why is this removed?

Reply:
I've updated version of `mimetype` which has those types buildin.